.env.example

```
@@ -33,6 +33,9 @@ REDIS_PASSWORD=
 ALLOW_CHARGE_VERSION_UPLOADS=
 TRIGGER_SROC_SUPPLEMENTARY=
 
+# Defaults to 20 if not set
+RECORDS_TO_DISPLAY_PER_PAGE=
```

We display different types of results across the service but this only applies to one screen. Therefore, could we be more specific about where this env var is used?

```suggestion
BILL_RUNS_TO_DISPLAY_PER_PAGE=
```

This will have a knock-on to the config file etc but rather than fill the review will noise I'll just highlight this here.

---

I shall sort that out